src/controllers/productController.js

```diff
@@ -16,9 +16,12 @@ const create = async (req, res) => {
     };
 
     const newProduct = await productService.create(product);
-    res.status(201).json(newProduct);
+    res.redirect('/api/v1/products');
 };
 
+const addProd = (req,res) => {
+    res.render('products/addProd');
+}
```

Comment:
Rename the function name such that it depicts the purpose what it does like `showNewForm`

Reply:
Done with this too, went for the same function name as stated by you.